src/servicebus/azext_servicebus/_params.py

```diff
@@ -0,0 +1,408 @@
+
+
+    # region Namespace Authorizationrule
+    for scope in ['sb namespace authorizationrule create',
+                  'sb namespace authorizationrule get',
+                  'sb namespace authorizationrule listkeys',
+                  'sb namespace authorizationrule regeneratekeys',
+                  'sb namespace authorizationrule delete']:
+        with self.argument_context(scope) as c:
+            c.argument('resource_group_name', arg_type=resource_group_name_type, required=True),
+            c.argument('namespace_name', options_list=['--namespace-name'], required=True,
+                       help='name of the Namespace'),
+            c.argument('authorization_rule_name', options_list=['--name'], required=True,
+                       help='name of the Namespace AuthorizationRule')
+
+
+    with self.argument_context('sb namespace authorizationrule create') as c:
+        c.argument('accessrights', options_list=['--access-rights'], required=False,
+                   help='Authorization rule rights of type list, allowed values are Send, Listen or Manage')
+
+    with self.argument_context('sb namespace authorizationrule regeneratekeys') as c:
+        c.argument('key_type', options_list=['--regeneratekey'], required=True,
+                   arg_type=get_enum_type(['PrimaryKey', 'SecondaryKey']),
+                   help='Authorization rule rights of type list, allowed values are Send, Listen or Manage')
+
+###################################################################
+    # region - Queue Create
+    with self.argument_context('sb queue create') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type, required=True)
+        c.argument('namespace_name', options_list=['--namespace-name'], required=True, help='name of the Namespace')
+        c.argument('queue_name', options_list=['--name', '-n'], required=True, help='Queue Name'),
+        c.argument('lock_duration', options_list=['--lock-duration'], validator=_validate_lock_duration,
+                   help='String ISO 8601 timespan duration of a peek-lock; that is, the amount of time that the message'
+                        ' is locked for other receivers. The maximum value for LockDuration is 5 minutes; the default'
+                        ' value is 1 minute.'),
+        c.argument('max_size_in_megabytes', options_list=['--max-size-in-megabytes'], type=int,
+                   validator=_validate_maxSizeInMegabytes,
+                   help='The maximum size of the queue in megabytes, which is the size of memory allocated for the'
+                        ' queue. Default is 1024.'),
+        c.argument('requires_duplicate_detection', options_list=['--requires-duplicate-detection'], type=bool,
+                   validator=_validate_requiresDuplicateDetection, help='A boolean value indicating if this'
+                                                                        ' queue requires duplicate detection.'),
```

Comment:
Make this a command line flag with `action='store_true'` instead of `type=bool`.

So then omission means false and inclusion of the param means true.

Reply:
resolved